content/management/configuration-api/v3.3/index.mdx

```diff
@@ -1310,6 +1348,26 @@ curl -X POST \
 
 <CodeResponse type="configuration" version="v3.3" json="getBot"/>
 
+```json
```

Comment:
This whole example should be removed: 
![image](https://user-images.githubusercontent.com/42515529/103542944-058f2a80-4e9e-11eb-942c-eb54d05e481a.png)

Instead, update the payload in payloads/management/v3.3/configuration-api/responses/bots/getBot.json